apps/cookbook/src/app/examples/example.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface Example {
```

Comment:
Added this so we can get type safety on these common example properties

Reply:
We already have that....?